src/main/java/com/minecolonies/coremod/colony/requestsystem/resolvers/StandardPlayerRequestResolver.java

```diff
@@ -136,6 +140,14 @@ public void resolve(@NotNull final IRequestManager manager, @NotNull final IRequ
         assignedRequests.add(request.getToken());
     }
 
+    private ITextComponent getRequestMessage(@NotNull final IRequest request)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "getRequestMessage" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)